index.js

```diff
@@ -2900,6 +2947,14 @@
 
   return S;
 
+  };
+
+  //  Export two versions of the Sanctuary module: one with type checking;
+  //  one without.
+  var S       = createSanctuary(true);
+  S.unchecked = createSanctuary(false);
+  return S;
+
```

Comment:
The current solution is similar to using native bindings with node-postgres: https://github.com/brianc/node-postgres#native-bindings. I can't think of any other modules that have an alternate version like this off the top of my head.

It might be nice to have plain string versions that you can import directly. Something like

`import S from 'sanctuary/alt/unchecked'`
`import S from 'sanctuary/alt/checked'` (for completeness sake, same as just writing `import S from 'sanctuary'`)

This of course also does not scale well, but is backwards compatible.

Nothing to do with sanctuary, but you could easily write a babel plugin that creates the appropriate import for you so you could just write something like

`import S from 'env-import:sanctuary'`

and the plugin would change it to the appropriate import line